js/store.js

```diff
@@ -1,59 +1,76 @@
 const store = {
-  init() {
-    this.addListeners();
+  _websites: null,
+
+  async init() {
+    const data = await browser.storage.local.get('websites');
+    this._websites = clone(data.websites);
+  },
+
+  async _write(websites) {
+    this._websites = clone(websites);
+
+    return await browser.storage.local.set({ websites });
   },
 
-  async getAll() {
-    return await browser.storage.local.get();
+  getAll() {
```

Comment:
@jkt thanks for the suggestions. `browser.storage.local.get()` is now called only during `init`. I have removed `_read()` . Feedback?! :)